devicon.json

```diff
@@ -7195,5 +7195,25 @@
                 "alias": "plain"
             }
         ]
+    },
+    {
+        "name": "postman",
+        "altnames": [],
+        "tags": [
+            "tool",
+            "testing"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark"
+            ],
+            "font": [
+                "plain",
+                "plain-wordmark"
+            ]
+        },
+        "color": "#ff6c37",
```

Comment:
This color should be `#f37036`
![image](https://user-images.githubusercontent.com/43886029/177012060-324cdbce-269c-4c9f-9133-affc4737d7e4.png)